assign_rights/serializers.py

```diff
@@ -30,6 +31,7 @@ class RightsShellSerializer(serializers.ModelSerializer):
     end_date = serializers.CharField()
     rights_granted = serializers.ListField(default=[])
     jurisdiction = serializers.CharField()
+    terms = serializers.CharField(source='license_terms')
```

Comment:
Any reason not to just migrate your model field? I know it's potentially more invasive, but now is the time to do it!